src/foraging_gui/MyFunctions.py

```diff
@@ -1176,8 +1176,8 @@ def _GetLaserWaveForm(self):
         # CLP, current laser parameter
         self.CLP_Color=eval('self.TP_Laser_'+N)
         self.CLP_Location=eval('self.TP_Location_'+N)
-        self.CLP_LaserPowerLeft=eval('self.TP_LaserPowerLeft_'+N)
-        self.CLP_LaserPowerRight=eval('self.TP_LaserPowerRight_'+N)
+        self.CLP_LaserPowerLeft=eval('self.TP_Laser1_power_'+N)
+        self.CLP_LaserPowerRight=eval('self.TP_Laser2_power_'+N)
```

Comment:
Whenever possible, avoid using eval(). getattr() is much better in places like this.

Reply:
Yes! Hi Han, please do not review this now. I plan to change the format and refactor some code. I will let you know when it's ready for review again.

Reply:
You can review the other PR first.  https://github.com/AllenNeuralDynamics/dynamic-foraging-task/pull/371

Reply:
Okay. You can turn this PR to Draft until it's ready